user/tests/wiring/watchdog/watchdog.cpp

```diff
@@ -178,6 +178,34 @@ test(WATCHDOG_07_notify_2) {
     assertEqual(magick, 0xdeadbeef);
 }
 
+#include "nrf_wdt.h"
+test(WATCHDDOG_08_reload_value_is_calculated_correctly) {
```

Comment:
Nit: I'd generalize this test probably and just move the platform-dependent functionality to get current reload value etc into a separate function.